.env_SAMPLE

```diff
@@ -0,0 +1,11 @@
+//export WORDPRESS=http://wordpress.domain
+//export GOVDELIVERY_BASE_URL=https://govdelivery.domain
+//export GOVDELIVERY_ACCOUNT_CODE=GODELIVERY_ACCT
+//export GOVDELIVERY_USER=godelivery@user.acct
+//export GOVDELIVERY_PASSWORD=P@$$w0rd
+//export SUBSCRIPTION_SUCCESS_URL=/govdelivery-subscribe-path
+//export SUBSCRIPTION_USER_ERROR_URL=/govdelivery-subscribe-error-path
+//export SUBSCRIPTION_SERVER_ERROR_URL=/govdelivery-subscribe-server-error-path
+//echo 'Environment variables for WORDPRESS, GOVDELIVERY, and SUBSCRIPTION are set'
```

Comment:
I don't understand the bit about SUBSCRIPTION. Looking at the internal docs they're just relative paths.

For the echo, that came from OAH but I didn't think much of it. Wouldn't terminal throw an error if it wasn't able to set the variable? We're not saying it's the correct variable, just that one was set.